internal/stack/record.go

```diff
@@ -91,67 +99,85 @@ func (c call) Record(opts ...recordOption) string {
 			opt(&optionsHolder)
 		}
 	}
-	name := runtime.FuncForPC(c.function).Name()
-	var (
-		pkgPath    string
-		pkgName    string
-		structName string
-		funcName   string
-		file       = c.file
-	)
+
+	name, file := extractName(c.function, c.file)
+	fnDetails := parseFunctionName(name)
+
+	return buildRecordString(optionsHolder, &fnDetails, file, c.line)
+}
+
+func extractName(function uintptr, file string) (name, fileName string) {
+	name = runtime.FuncForPC(function).Name()
 	if i := strings.LastIndex(file, "/"); i > -1 {
-		file = file[i+1:]
+		fileName = file[i+1:]
+	} else {
```

Comment:
Once more point:
record.go was 100% coveraged before refactor and 98.8% after.

One branch of the code not tested: point when `file` doesn't contain `/`.
Add test for this please and PR will be merge.

Reply:
I fixed the issue by replace the code by function from standard library - for extrace file name from path.